lib/app/routes/exercises.rb

```diff
@@ -72,6 +72,10 @@ class Exercises < Core
         flash[:success] = "#{exercise.problem.name} in #{exercise.problem.track_id} is now reactivated."
         redirect '/'
       end
+
+      get '/five_a_aday/:key' do |key|
```

Comment:
Is this the user's key? Is this used by some javascript? If so, let's put it in the API